src/Robo/Plugin/Commands/ValidateConfigCommands.php

```diff
@@ -50,7 +50,7 @@ public function validateDrupalConfig($siteDir = 'default'): void
             ->run();
         $drushOutput = trim($result->getOutputData());
         $configJson = json_decode($drushOutput);
-        if (!empty($configJson)) {
+        if (!is_null($configJson)) {
```

Comment:
I am tempted to suggest a `try{}catch{}` around `json_decode()` unless you are guaranteed that `getOutputData()` will return json-decodable output no matter what happens in the task call.  Maybe getOutputData already does that and guarantees JSON in all cases, in which case, please ignore the above.

_But_ if there is a contract  as to what getOutputData will return, I would assume that one of the things in that output data json blob is going to be a success/error state. If that is the case, it seems it would be better to test that value. ie `if ($configJson->status == 0)`